lib/jekyll/tags/include.rb

```diff
@@ -12,17 +12,23 @@ def initialize(msg, path)
     end
 
     class IncludeTag < Liquid::Tag
-      VALID_SYNTAX = /([\w-]+)\s*=\s*(?:"([^"\\]*(?:\\.[^"\\]*)*)"|'([^'\\]*(?:\\.[^'\\]*)*)'|([\w\.-]+))/
-      VARIABLE_SYNTAX = /(?<variable>[^{]*(\{\{\s*[\w\-\.]+\s*(\|.*)?\}\}[^\s{}]*)+)(?<params>.*)/
+      VALID_SYNTAX = /
+        ([\w-]+)\s*=\s*
+        (?:"([^"\\]*(?:\\.[^"\\]*)*)"|'([^'\\]*(?:\\.[^'\\]*)*)'|([\w\.-]+))
+      /x
+      VARIABLE_SYNTAX = /
+        (?<variable>[^{]*(\{\{\s*[\w\-\.]+\s*(\|.*)?\}\}[^\s{}]*)+)
+        (?<params>.*)
+      /x
 
       def initialize(tag_name, markup, tokens)
         super
         matched = markup.strip.match(VARIABLE_SYNTAX)
         if matched
-          @file = matched['variable'].strip
-          @params = matched['params'].strip
+          @file = matched["variable"].strip
+          @params = matched["params"].strip
         else
-          @file, @params = markup.strip.split(' ', 2)
+          @file, @params = markup.strip.split(" ", 2)
```

Comment:
This might be a good time to fix that bad split.  Users are prone to alignment because it's prettier, if you don't mind can you make this `markup.strip.split(/\s+/, 2)` that way users can infinity space and align any way they see fit.  I don't know if it's worth it though considering it's not been a problem before?